app/containers/EpisodeAdminEditor/reducer.js

```diff
@@ -6,15 +6,45 @@
 
 import { fromJS } from 'immutable';
 import {
-  DEFAULT_ACTION,
+  UPDATE_EPISODE_PENDING,
+  UPDATE_EPISODE_SUCCESS,
+  UPDATE_EPISODE_FAILED,
+  DELETE_EPISODE_PENDING,
+  DELETE_EPISODE_SUCCESS,
+  DELETE_EPISODE_FAILED,
 } from './constants';
 
-const initialState = fromJS({});
+const initialState = fromJS({
+  loading: false,
+  error: false,
+});
 
 function episodeAdminEditorReducer(state = initialState, action) {
   switch (action.type) {
-    case DEFAULT_ACTION:
-      return state;
+    case UPDATE_EPISODE_PENDING:
+      return state
+      .set('loading', true)
```

Comment:
Use the spread operator?

```
return {
    ... state,
    loading: true,
    error: false
}
```

Or is it because of immutable? I'd rather just create a new object, as one of the mian points of Redux is that the state is completely new for each action.